### PR TITLE
do apt-get update before apt install; cd into src before running script

### PR DESCRIPTION
**Pull request by mkinney**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,9 +66,11 @@ The section below is for Ubuntu and other Debian based distros.
 
 ```
 # Resolve build dependencies
+sudo apt-get update
```
Comment (ivandavidov) [inline README.md]: I'd rather keep ``apt`` instead of introducing ``apt-get``.

File: README.md [hunk 2]
```diff
@@ -66,9 +66,11 @@ The section below is for Ubuntu and other Debian based distros.
...
 sudo apt install wget make gawk gcc bc bison flex xorriso libelf-dev libssl-dev
 
 # Build everything and produce ISO image.
+cd src
```
Comment (ivandavidov) [inline README.md]: Have you checked the release archives before introducing your change? I believe there is no ``src`` folder there.

File: README.md (hunk 1 above)
Comment (mkinney) [reply README.md]: No worries.

File: README.md (hunk 2 above)
Comment (mkinney) [reply README.md]: There is when you do a 'git clone' No worries. I'll close the PR.

File: README.md (hunk 1 above)
Comment (scoopgracie) [reply README.md]: Whenever you have `apt`, you have `apt-get`. In fact, everything should switch over to `apt-get`, which has a stable CLI interface suitable for scripts.

Comment (scoopgracie) [reply README.md]: And for putting in documentation. `apt`'s interface could have spontaneous breaking changes.

**Pull request closed without merging**